.circleci/config.yml

```diff
@@ -29,7 +29,8 @@ jobs:
           command: |
             python3 -m venv venv
             . venv/bin/activate
-            pip install --upgrade pip setuptools wheel twine
+            pip install --upgrade pip
```

Comment:
@miguelbalmeida  making this change I have here fixed the build step your PR is breaking on, however it doesn't fix the next step where I think it tries to overwrite something in pypi that it shouldn't...

Reply:
@sesprit Sorry, I only saw your comment on my PR and not this one. Indeed I made this change in my PR also, by splitting the original command of `pip install --upgrade pip setuptools wheel twine` into first upgrading pip, and then installing the other dependencies (which will use the upgraded pip to install them, rather than whatever is on the Docker image).

I don't particularly like that a PR tries to deploy something on PyPI. For example, when we do change the version to 3.7.2, the deployment on PyPI will happen when we create the PR, not when we merge it to master. IMO that's very confusing and not a good practice.

I pinged you on a Jira ticket where we're discussing how to improve on this, among other improvements to snowconn.

Reply:
I agree!